The NETRC environment variable is honoured (#32007)
hashicorp/terraform: website/docs/language/modules/sources.mdx

using one of the forms documented elsewhere on this page.

If an HTTP/HTTPS URL requires authentication credentials, use a `.netrc`

> to configure these. By default, Terraform will search for the `.netrc` file 

Author: ```suggestion
file to configure these. By default, Terraform will search for the `.netrc` file 
```

in your HOME directory. However, that default filesystem location can be overriden 

> through the `NETRC` environment variable. For information on the `.netrc` format,

Author: ```suggestion
by setting the `NETRC` environment variable. For information on the `.netrc` format,
```

> file to configure these. By default, Terraform will search for the `.netrc` file 
> in your HOME directory. However, that default filesystem location can be overriden 
> by setting the `NETRC` environment variable. For information on the `.netrc` format,
> see [the documentation for using it in `curl`](https://everything.curl.dev/usingcurl/netrc).

Reviewer: ```suggestion
file to configure the credentials. By default, Terraform searches for the `.netrc` file 
in your HOME directory. However, you can override the default filesystem location by setting the `NETRC` environment variable. For information on the `.netrc` format,
refer to [the documentation for using it in `curl`](https://everything.curl.dev/usingcurl/netrc).
```

Suggest changing to "the credentials" for clarity and to avoid "These" without a subject. Suggest changing "will search" to "searches" to avoid a future tense style issue. Suggest changing to "you can override" to avoid a passive voice style issue. Suggest changing "see" to "refer to" to avoid ableist language.
